docs/LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2022-2025 just-the-docs
+Copyright © 2022-2025 just-the-docs
```

Comment:
Probably good to blanket ignore the license files so we don't accidentally change the wording.

Reply:
Have pushed a way to do this